.github/workflows/deploy-docs-site.yml

```diff
@@ -29,22 +36,47 @@ jobs:
           cache: "yarn"
           cache-dependency-path: docusaurus
 
+
       - name: Run Docusaurus
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./tools/bin/deploy_docusaurus
+        run: |-
+          export SKIP_DEPLOY="no"
+          if [ "${{github.event_name}}" = "pull_request" ]; then
+              export SKIP_DEPLOY="yes"
+          fi
```

Comment:
can we invert this logic?  I don't expect this to fail, but I think it would be safer to have `SKIP_DEPLOY=yes` as our default.  
The way this is currently written if we were to find an unintended input for `${{github.event_name}}` we would deploy and that feels more risky

Reply:
ok, _corrected in_ [a4300bb](https://github.com/airbytehq/airbyte/pull/20563/commits/a4300bbbe0af81e3d12f91039c49ace3a2d33bb7)